index.rst

```diff
@@ -46,14 +46,314 @@
 
 .. note::
 
-   **This technote is not yet published.**
+   When deploying the Telescope and Site systems for the main telescope at NCSA we started to experience severe startup slow downs.
```

Comment:
"startup slow downs" --> slow downs when starting up components. I'd also put in a rough time estimate.